### PR TITLE
develop

### PR DESCRIPTION
**Pull request by Layrovell**
[DEMO LINK](https://Layrovell.github.io/react_goods-selector/)

### CODE REVIEW

**Review by Serhii-Alieksieievych — APPROVED** [review]
не впевнений що так має бути, але в вимогах це прямо не забороняється, тому апруваю)
https://prnt.sc/x5kdpd

**Review by varseniuk — CHANGES_REQUESTED** [review]
Serhii is right. You should add only one instance of any good at a time. Fix it. Everything else is ok

File: src/components/Selector/Selector.jsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+export class Selector extends React.Component {
+  state = {
+    selectedGoods: [],
+  };
+
+  addItem = (item) => {
+    this.setState(prevState => ({
+      selectedGoods: [...prevState.selectedGoods, item],
+    }));
+  };
```
Comment (varseniuk) [inline src/components/Selector/Selector.jsx:14]: Your function still adds multiple same items to the list. You can prevent this by adding check here like
`if(!this.state.selectedGoods.includes(item)) {...}`

**Review by varseniuk — CHANGES_REQUESTED** [review]
Still can add multiple items

**Review by varseniuk — APPROVED** [review]
👍

**Pull request closed without merging**